app/org/zalando/markscheider/MetricsPlugin.scala

```diff
@@ -72,7 +72,7 @@ class MetricsPlugin @Inject() (
 
     if (enabled) {
       val rateUnit = configuration.getString("org.zalando.markscheider.rateUnit", validUnits).getOrElse("SECONDS")
-      val durationUnit = configuration.getString("org.zalando.markscheider.durationUnit", validUnits).getOrElse("MILLISECONDS")
+      val durationUnit = configuration.getString("org.zalando.markscheider.durationUnit", validUnits).getOrElse("SECONDS")
```

Comment:
I see. I wanted to make it consistent with the defaults from metrics-play but will change it back.